backend/usage_v2/migrations/0001_initial.py

```diff
@@ -0,0 +1,144 @@
+# Generated by Django 4.2.1 on 2024-07-29 09:16
+
+import uuid
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ("account_v2", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Usage",
```

Comment:
@muhammad-ali-e Suggest to keep the same name for the ORM model and corresponding db table.

Reply:
renamed db table name to usage, Hence both model and table name have same name.